ui/partials/perf/comparechooserctrl.html

```diff
@@ -21,8 +21,8 @@
           <select id="new-project-selector" class="form-control" ng-model="newProject" ng-options="project.name for project in projects"/>
           <label for="new-revision-input">Revision</label>
             <div class="form-group has-error">
-            <input id="new-revision-input" maxlength="12" class="form-control" ng-model="newRevision">
-            <br/>
+              <input id="new-revision-input" maxlength="12" class="form-control" ng-model="newRevision">
+              <br/>
```

Comment:
Please see my comments in https://github.com/mozilla/treeherder/pull/588#discussion_r31883014 regarding this tag.